orderer/consensus/etcdraft/consenter_test.go

```diff
@@ -41,6 +42,17 @@ import (
 	"go.uber.org/zap/zapcore"
 )
 
+var (
+	sampleCert = "LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUJxakNDQVZHZ0F3SU" +
```

Comment:
In general, if we're going to have complex binary literals, at least a comment mentioning how they were generated would be appreciated.

Reply:
Removed it